tests/RobotFramework/tests/cumulocity/firmware/firmware_operation_workflow.robot

```diff
@@ -0,0 +1,34 @@
+*** Settings ***
+Resource            ../../../resources/common.resource
+Library             ThinEdgeIO
+Library             Cumulocity
+
+Suite Setup         Custom Setup
+Test Setup          Custom Test Setup
+Test Teardown       Get Logs
+
+Test Tags           theme:tedge_agent
+
+
+*** Test Cases ***
+Send firmware update operation from Cumulocity IoT
+    Cumulocity.Should Contain Supported Operations    c8y_Firmware
+    ${operation}=    Cumulocity.Install Firmware    tedge-core    1.0.0    https://abc.com/some/firmware/url
+    ${operation}=    Cumulocity.Operation Should Be SUCCESSFUL    ${operation}
+    Cumulocity.Device Should Have Firmware    tedge-core    1.0.0    https://abc.com/some/firmware/url
+
+*** Keywords ***
+Custom Test Setup
+    Execute Command
+    ...    cmd=echo 'tedge ALL = (ALL) NOPASSWD: /usr/bin/tedge, /usr/bin/systemctl, /etc/tedge/sm-plugins/[a-zA-Z0-9]*, /bin/sync, /sbin/init, /sbin/shutdown, /usr/bin/on_shutdown.sh, /usr/bin/tedge-write /etc/*' > /etc/sudoers.d/tedge
+
+Custom Setup
+    ${DEVICE_SN}=    Setup
+    Set Suite Variable    $DEVICE_SN
+    Device Should Exist    ${DEVICE_SN}
+
+    Execute Command    tedge config set c8y.enable.firmware_update true
```

Comment:
Wondering why we've disabled this by default at the mapper level. Unless someone explicitly creates a C8Y operation file or defines a workflow, the supported operation won't be declared by the mapper. So, I'm not sure why we want to disable the whole mapping itself, by default. @rina23q do you remember why? Can we switch to back to `true`?

Reply:
I believe it was done to ensure we don't break systems which are using `c8y-firmware-plugin`...though we do need to think about a cleaner upgrade path (though correct me if I'm wrong here).

Reply:
I think first mapper implementation was done while agent side was not completed. That's why it was marked `false` as a temporary solution, however, it's forgotten to be `true`, even after firmware implementation in agent side was done...

Reply:
Resolved by bdaabc5